build/scripts/tests.sh

```diff
@@ -82,7 +81,7 @@ do
     fi
 
     # https://github.com/dotnet/roslyn/issues/29380
-    if ${runner} "${xunit_console}" "${file_name[@]}" -xml "${log_file}" -parallel none -maxThreads 2
```

Comment:
The use of `-maxThreads 2` was a testing mistake that made it into the last PR. Given we aren't running in parallel this really has no effect.